zerver/tests/test_auth_backends.py

```diff
@@ -5207,6 +5273,14 @@ def test_login_success_when_user_does_not_exist_with_split_full_name_mapping(sel
             self.assertEqual(user_profile.full_name, "First Last")
             self.assertEqual(user_profile.realm.string_id, "zulip")
 
+    def test_do_set_ldap_uid(self) -> None:
+        user_profile = self.example_user("hamlet")
+        user_profile.ext_auth_uid = None
+        do_set_ldap_unique_identifier(user_profile, "ou=test,dc=dc=zulip,dc=com")
```

Comment:
this test should use a valid `dn` for users - `ou=test,dc=dc=zulip,dc=com` can't be that (no uid or similar attribute included)

Reply:
Don't know in what parallel universe I was when writing that...
Should be fixed now.